xbmc/GUIInfoManager.h

```diff
@@ -1,11 +1,3 @@
-/*!
-\file GUIInfoManager.h
-\brief
-*/
-
-#ifndef GUIINFOMANAGER_H_
-#define GUIINFOMANAGER_H_
-
```

Comment:
`#pragma once` if you want get rid of this include guard

Reply:
done, thanks